CHANGELOG.md

```diff
@@ -4,6 +4,122 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.10.1rc1] - 2021-07-21: "<awaiting name selection>"
+
+This release named by @nalinbhardwaj.
+
```

Comment:
*NOTE*: dual funding and offers protocols have changed, and are incompatible with older releases (they're both still draft)